### ISSUE TITLE
NullPointerException On Track Gesture Event

### ISSUE THREAD

**Issue by bleege** [mapbox/mapbox-gl-native#4337]
```java
Android: 5.0.1
Manufacturer: LGE
Model: LG-D850
CrashReporter Key: 0793E332-AB82-CA2C-589E-E6631578332246880CFE
Date: Thu Mar 10 06:42:29 GMT+08:00 2016

java.lang.NullPointerException: Attempt to invoke virtual method 'void com.mapbox.mapboxsdk.telemetry.MapboxEventManager.pushEvent(java.util.Hashtable)' on a null object reference
	at com.mapbox.mapboxsdk.maps.MapView.trackGestureEvent(Unknown Source)
	at com.mapbox.mapboxsdk.maps.MapView.access$1900(Unknown Source)
	at com.mapbox.mapboxsdk.maps.MapView$RotateGestureListener.onRotateBegin(Unknown Source)
	at com.almeros.android.multitouch.gesturedetectors.RotateGestureDetector.handleStartProgressEvent(Unknown Source)
                                                                          resetState
	at com.almeros.android.multitouch.gesturedetectors.BaseGestureDetector.onTouchEvent(Unknown Source)
                                                                        handleStartProgressEvent
                                                                        resetState
	at com.mapbox.mapboxsdk.maps.MapView.onTouchEvent(Unknown Source)
	at android.view.View.dispatchTouchEvent(View.java:8400)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2425)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2159)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2431)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2173)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2431)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2173)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2431)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2173)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2431)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2173)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2431)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2173)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2431)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2173)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2431)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2173)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2431)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2173)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2431)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2173)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2431)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2173)
	at com.android.internal.policy.impl.PhoneWindow$DecorView.superDispatchTouchEvent(PhoneWindow.java:2391)
	at com.android.internal.policy.impl.PhoneWindow.superDispatchTouchEvent(PhoneWindow.java:1741)
	at android.app.Activity.dispatchTouchEvent(Activity.java:2826)
	at android.support.v7.view.WindowCallbackWrapper.dispatchTouchEvent(Unknown Source)
	at com.android.internal.policy.impl.PhoneWindow$DecorView.dispatchTouchEvent(PhoneWindow.java:2352)
	at android.view.View.dispatchPointerEvent(View.java:8590)
	at android.view.ViewRootImpl$ViewPostImeInputStage.processPointerEvent(ViewRootImpl.java:4098)
	at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:3964)
	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:3488)
	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:3541)
	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:3507)
	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:3515)
	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:3488)
	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:3541)
	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:3507)
	at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:3617)
	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:3515)
	at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:3674)
	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:3488)
	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:3541)
	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:3507)
	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:3515)
	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:3488)
	at android.view.ViewRootImpl.deliverInputEvent(ViewRootImpl.java:5789)
	at android.view.ViewRootImpl.doProcessInputEvents(ViewRootImpl.java:5763)
	at android.view.ViewRootImpl.enqueueInputEvent(ViewRootImpl.java:5734)
	at android.view.ViewRootImpl$WindowInputEventReceiver.onInputEvent(ViewRootImpl.java:5912)
	at android.view.InputEventReceiver.dispatchInputEvent(InputEventReceiver.java:185)
	at android.os.MessageQueue.nativePollOnce(Native Method)
	at android.os.MessageQueue.next(MessageQueue.java:143)
	at android.os.Looper.loop(Looper.java:122)
	at android.app.ActivityThread.main(ActivityThread.java:5292)
	at java.lang.reflect.Method.invoke(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:372)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:908)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:703)
```

**Comment by tobrun** [comment]
This was fixed with #4266

**Comment by bleege** [comment]
Closing then!
